includes/options/views/class-amp-analytics-options-submenu-page.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/options/views/class-amp-analytics-options-serializer.php' );
+
+class AMP_Analytics_Options_Submenu_Page {
+
+	private function render_option($id = "", $type = "", $config = "") {
+			?>
+			<div class="analytics-data-container">
+				<form method="post" action="<?php echo esc_html( admin_url( 'admin-post.php' ) ); ?>">
+					<h2><?php echo __( 'Analytics Component', 'amp' ) ?>: <?php echo ($type ? $type . ':' : '') . substr($id, -6) ?></h2>
+					<div class="options">
+						<p>
+							<label><?php echo __( 'Type', 'amp' ) ?>: </label>
+							<input class="option-input" type="text" name=vendor-type value="<?php echo esc_attr( $type ); ?>" />
+							<label><?php echo __( 'Id', 'amp' ) ?>: </label>
+							<input type="text" name=id value="<?php echo substr(esc_attr( $id ), -6); ?>" readonly />
+                            <input type="hidden" name=id-value value="<?php echo $id; ?>" />
+						</p>
+						<p>
+							<label><?php echo __( 'JSON Configuration', 'amp' ) ?>:</label>
+							<br />
+							<textarea rows="10" cols="100" name="config"><?php echo esc_textarea( $config ); ?></textarea>
+						</p>
+						<input type="hidden" name="action" value="analytics_options">
+					</div><!-- #analytics-data-container -->
+				<p>
+			<?php
+			    wp_nonce_field( 'analytics-options', 'analytics-options');
+				submit_button('Save', 'primary', 'save', false);
+				submit_button('Delete', 'delete button-primary', 'delete', false);
+				?>
+				</p>
+				</form>
+			</div><!-- .wrap -->
+			<?php
+	}
+
+	public function render_title() {
+		?>
+		<div class="wrap">
+			<h1><?php echo esc_html( get_admin_page_title() ); ?></h1>
+		<?php
+
+		// If redirected from serializer, check if action succeeded
+		global $_GET;
+		if ( isset( $_GET['valid'] ) ) {
+            if ( ! esc_attr( $_GET['valid'] ) ) {
+                ?>
+                <h3 style='color:red'><?php echo 'Action not taken: invalid input!'; ?></h3>
```

Comment:
We should use WordPress standard admin notices here: https://digwp.com/2016/05/wordpress-admin-notices/